cli/types/cypress.d.ts

```diff
@@ -2138,6 +2138,22 @@ declare namespace Cypress {
     ```
      */
     writeFile<C extends FileContents>(filePath: string, contents: C, options?: Partial<WriteFileOptions>): Chainable<C>
+    /**
+     * Write to a file with the specified encoding and contents.
+     *
+     * The `encoding` option in `options` precedes the `encoding` argument.
+     *
+     * @see https://on.cypress.io/writefile
+    ```
+    cy.writeFile('path/to/ascii.txt', 'Hello World', 'utf8', {
+      flag: 'a+',
+      encoding: 'ascii'
```

Comment:
I'd probably remove the double encoding from the example because I wouldn't exactly encourage this.

```suggestion
```